tests/test_plot.py

```diff
@@ -267,26 +272,12 @@ def test_kde_plot_1d(plot_1d):
         line, = plot_1d(ax, data, cmap=plt.cm.Blues)
         assert line.get_color() == plt.cm.Blues(0.68)
 
-        # Check xmin
-        xmin = -0.5
-        line, = plot_1d(ax, data, xmin=xmin)
-        assert (line.get_xdata() >= xmin).all()
-
-        # Check xmax
-        xmax = 0.5
-        line, = plot_1d(ax, data, xmax=xmax)
-        assert (line.get_xdata() <= xmax).all()
-
-        # Check xmin and xmax
-        line, = plot_1d(ax, data, xmin=xmin, xmax=xmax)
-        assert (line.get_xdata() <= xmax).all()
-        assert (line.get_xdata() >= xmin).all()
-        plt.close("all")
-
```

Comment:
Always makes me uncomfortable to see tests deleted, but here we can!

Reply:
Yes, I agree. And it might mean encountering issues further along, but we are fundamentally changing the way we are handling data vs plotting limits, and these tests no longer cover that.